app/importers/open_access_button_publication_importer.rb

```diff
@@ -24,7 +24,7 @@ def import_new
   private
 
   def all_pubs
-    Publication.where.not(doi: nil).where.not(doi: '').where(%{open_access_url IS NULL OR open_access_url = ''})
+    Publication.where.not(doi: nil).where.not(doi: '')
```

Comment:
@nicktiberi Do the tests break if you put the old sql statement back in? I'm seeing that you changed the tests, but I don't see the underlying test data has changed. I'm wondering if we could get a regression bug here.

Reply:
Yes, they do. The tests have some code that populates the publication with an open access URL:

```
before { pub.update_attribute(:open_access_url, "existing_url") }
```

With the deleted WHERE clause put back, these tests would fail because `all_pubs` would not include the test publication, since it would have a populated `open_access_url`.